.github/workflows/ci.yaml

```diff
@@ -125,7 +129,7 @@ jobs:
         run: |
           export GOPATH=$(go env GOPATH)
           export PATH=${PATH}:$GOPATH/bin
-          make verify lint binary test test/integration
+          make verify binary test test/integration
```

Comment:
Why the move from using `lint` target to using the github action?

Reply:
The lint target still works and can be used on the CLI, I though that the github action was a better choice because it is integrated with GITHUB and it is easier/faster by looking at the `File changed` tab to see where the violation is.

Do you see any drawback in using the github action?

Reply:
No, I do not see any major drawback.
We might have to remember updating the version so that it aligns to whatever version we are using locally when a change happen. 
I agree seeing the violations directly in the UI  is better than having to see the logs. But ideally, that shouldn't be needed if we (the team) during dev, proactively make sure that the linting checks are intact.

Reply:
@ziccardi following on to what I said above , it'll be good to have linting on pre-commit hook: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/.githooks/pre-commit. What do you think? Is there a task for this?

Reply:
I will add a task for that